index.js

```diff
@@ -23,68 +26,72 @@ function getRandomNumber(min, max) {
 }
 
 function checkGuess() {
-  // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+  if (guess < guessInput.min) {
+    wayTooLow.style.display = "block";
+  } else if (guess > guessInput.max) {
+    wayTooHigh.style.display = "block";
+  } else if (guess === targetNumber) {
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
-    correctMessage.style.display = '';
-
+    correctMessage.style.display = "";
     submitButton.disabled = true;
     guessInput.disabled = true;
-  }
 
-  if (guess !== targetNumber) {
+    if (attempts === maxNumberOfAttempts) {
+      submitButton.disabled = true;
+      guessInput.disabled = true;
+    }
```

Comment:
When you were refactoring your code, you accidentally removed any place where the "you lose" message displays. 

It should be unhidden here to show the game loss state

Reply:
you should also hide the "too high" or "two low" message when the game is over, or maybe we would modify it so it just doesn't include the "guess again?"

<img width="264" alt="Screen Shot 2023-09-16 at 8 28 05 PM" src="https://github.com/CristianMoran1/intro-to-programming-section-5-1/assets/12771651/80212892-c7d8-4730-a587-22630e23d20e">